mobile/src/screens/marketplace.js

```diff
@@ -473,6 +476,36 @@ class MarketplaceScreen extends PureComponent {
     )
   }
 
+  injectDeviceFingerprint = async () => {
+    const screenResolution = `${Math.round(
+      Dimensions.get('window').width
+    )}x${Math.round(Dimensions.get('window').height)}`
+
+    const fingerprint = {
+      deviceId: DeviceInfo.getDeviceId(),
```

Comment:
Don't those getters return a promise? should we await or looks like the library also provides synchronous methods (same method name but postfix with Sync).
https://github.com/react-native-community/react-native-device-info

Reply:
@franckc Not everything returns a Promise, I used await on ones that do return Promise. https://github.com/react-native-community/react-native-device-info#api

Reply:
ah yes indeed your are right! thanks